src/components/__test__/quotes.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import renderer from 'react-test-renderer';
+import Quotes from '../quotes';
+
+it('should render the quotes component', () => {
+  const wrapper = ReactDOM.createRoot(document.createElement('div'));
+  wrapper.render(<Quotes />);
+});
+
+it('should match the snapshot', () => {
+  const tree = renderer.create(<Quotes />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Also make sure there is the usage of **React Testing Library** to simulate user interaction for as required for the project.

Reply:
Hello code reviewer! 
In this file "Quote page", we can't apply any user interaction. Quote page data come from an API, it doesn't have user interaction.

We use the user Interaction and the React Testing Library on the "Calculator page" simulating the clicks on the buttons and looking at the answers on the screen.